### PR TITLE
BUG max-lr not used correctly

### PR DESCRIPTION
**Pull request by GeorgWa**
The maximum learning rate was not correctly read from the settings.

### CODE REVIEW

File: alphadia/transferlearning/train.py [hunk 1]
```diff
@@ -547,7 +547,7 @@ def finetune_ms2(
             epoch=self.settings["epochs"],
             batch_size=self.settings["batch_size"],
             warmup_epoch=self.settings["warmup_epochs"],
-            lr=settings["max_lr"],
+            lr=self.settings["max_lr"],
```
Comment (mschwoer) [inline alphadia/transferlearning/train.py:550]: what's the role of this `settings` object that this code was accidentially referring to?
could it be removed?
in code, it's referenced only here:
```python
self.reduce_lr_on_plateau = torch.optim.lr_scheduler.ReduceLROnPlateau(
            optimizer,
            mode="min",
            patience=settings["lr_patience"],
            factor=0.5,
        )
```
(and in one notebook)

if it is to be kept: should we rename it? add some docstring?

Comment (GeorgWa) [reply alphadia/transferlearning/train.py:550]: it shouldn't be referenced anywhere :D good point

Comment (GeorgWa) [reply alphadia/transferlearning/train.py:550]: @mo-sameh Can we get rid of the settings object and use kwargs with defaults?

Comment (mschwoer) [reply alphadia/transferlearning/train.py:550]: now as far as I see the only usage left is in `finetuning.ipynb` .. we could just move it there?

Comment (GeorgWa) [reply alphadia/transferlearning/train.py:550]: Will be completely moved to kwargs 👍 by mo

**Review by mschwoer — APPROVED** [review]
